tests/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 #Copyright (c) Microsoft. All rights reserved.
 #Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+set(build_yet FALSE)
```

Comment:
Confused about what this is ...?

Reply:
Yeah, I will document it better.

We produce .exe and .dll from these tests, but realistically for the purpose of preprocessing in the IDE, only 1 is needed (the exe). So this acts as a "singleton" to not build the dll test.